lib/modules/datasource/glasskube-packages/index.ts

```diff
@@ -0,0 +1,75 @@
+import { cache } from '../../../util/cache/package/decorator';
+import { joinUrlParts } from '../../../util/url';
+import * as glasskubeVersioning from '../../versioning/glasskube';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+import {
+  GlasskubePackageManifestYaml,
+  GlasskubePackageVersions,
+  GlasskubePackageVersionsYaml,
+} from './schema';
+
+export class GlasskubePackagesDatasource extends Datasource {
+  static readonly id = 'glasskube-packages';
+  static readonly defaultRegistryUrl =
+    'https://packages.dl.glasskube.dev/packages';
+
```

Comment:
```suggestion
  override readonly customRegistrySupport = true;
```
Activate custom registry support.